pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "danger-python"
+version = "0.0.1"
+description = "Write your Dangerfiles in Python."
+authors = ["Jakub Turek <jkbturek@gmail.com>"]
```

Comment:
I'm cool with this, but I'd like to be able to ship releases also, once that's happening please (it's so I can feel good about conforming to [this](https://github.com/Moya/contributors/blob/master/Contributing.md))

Reply:
I've forgotten about it, this was autogenerated by poetry. Are you sure we shouldn't make it more community-centric (like Danger Systems)? I am happy to change it 👍

Reply:
The working I tend to use is: " orta therox, f-meloni, and the Danger org contributors"

So maybe `["Jakub Turek <jkbturek@gmail.com>", "Orta Therox <danger@orta.io>", "Danger org contributors"]` ?

Reply:
I switched the order because I didn't really do any work here so far, at least on Swift I spent about 6 months solo before f-meloni rode in to save the day